prime-router/src/main/resources/jsonTemplate.json

```diff
@@ -0,0 +1,37 @@
+{
+  "mdc": {
```

Comment:
Is this getting mapped to `customDimensions` automatic with the java agent?

Reply:
Yup thats where it ends up.